### PR TITLE
Add error checking on project page

### PR DESCRIPTION
**Pull request by Pratheekl22**

### CODE REVIEW

**Comment by lizcooper44** [comment]
Can you make this similar to the team creation page where it displays which fields have errors?

**Comment by lizcooper44** [comment]
Also the disable submit button thing same as teams creation page would be great!

**Comment by Pratheekl22** [comment]
Yep I added everything. Do you guys think its a good idea to say what input is expected in the helper text? Or is it too much?

**Review by dosorno7 — APPROVED** [review]
lgtm

File: react-app/src/components/CreateProjectModal/createProjectModal.tsx [hunk 1]
```diff
@@ -40,29 +45,74 @@ export default function CreateProjectModal( {getCreateProjectInfo}: any ) {
...
         clientName: string,
         activeStatus: string) => {
-
-        getCreateProjectInfo(teamNumber, section, organization, clientName, activeStatus); 
-        handleClose();
+        
+        getCreateProjectInfo(teamNumber, section, organization, clientName, activeStatus);
+        if (teamNumber.length == 0 || section.length == 0 || organization.length == 0 || clientName.length == 0|| activeStatus.length == 0) {
+            console.warn("Inputs are wrong");
+            //{faultyInput ? 'Invalid Inputs'  : ''}
```
Comment (JuanNRamirez) [inline react-app/src/components/CreateProjectModal/createProjectModal.tsx:52]: Nit: Remove unnecessary commented line

**Review by JuanNRamirez — APPROVED** [review]
LGTM apart from one small NIT comment.

**Pull request merged**